tests/apitests/features/api_tests.feature

```diff
@@ -0,0 +1,120 @@
+@api
+Feature: Newsfeed API tests
+
+  Scenario: Check microservice status
+    When get request to "status/" is received
+    Then response status should be 200
+    And response should contain "OK" status
+
+  Scenario: Verify documentation
+    When get request to "docs/" is received
+    Then response status should be 200
+    And response should contain documentation
+
+  Scenario: Get empty result if there are no events
+    When get request to "newsfeed/123/events/" is received
+    Then response status should be 200
+    And response should contain empty result
+
+  Scenario: Get events
+    Given post request to "newsfeed/123/events/" is received with body:
+      | key       | value      |
+      | field_1   | some_data  |
+      | field_2   | other_data |
+      | field_etc | etc_data   |
+    When get request to "newsfeed/123/events/" is received
+    Then response status should be 200
+    And response should contain results list
+
+  Scenario: Add events
+    When post request to "newsfeed/123/events/" is received with body:
+      | field_1   | some_data  |
+      | field_2   | other_data |
+      | field_etc | etc_data   |
+    Then response status should be 202
+    And response should include the event
+
+  Scenario: Delete event
+    Given post request to "newsfeed/123/events/" is received with body:
+      | field_1   | some_data  |
+      | field_2   | other_data |
+      | field_etc | etc_data   |
+    When delete request to "newsfeed/123/events/{event_id}/" is received
+    Then response status should be 204
+    And this event should be absent
+
+  Scenario: Get subscriptions
+    Given post request to "newsfeed/123/subscriptions/" is received with body:
+      | to_newsfeed_id | 124 |
+    When get request to "newsfeed/123/subscriptions/" is received
+    Then response status should be 200
+    And response should contain results list
+
+  Scenario: Add subscriptions
+    When post request to "newsfeed/123/subscriptions/" is received with body:
+      | to_newsfeed_id | 124 |
+    Then response status should be 201
+    And response should include the subscription
+
+  Scenario: Delete subscription
+    Given post request to "newsfeed/123/subscriptions/" is received with body:
+      | to_newsfeed_id | 124 |
+    When delete request to "newsfeed/123/subscriptions/{subscription_id}/" is received
+    Then response status should be 204
+    And this subscription should be absent
+
+  Scenario: Get subscribers
```

Comment:
Please, add `Given` that newsfeed `1213` has subscribers (`Subscription` object `to_newsfeed_id = 1213`).